app/assets/javascripts/initializers/initializeBaseTracking.js.erb

```diff
@@ -15,7 +15,6 @@ function initializeBaseTracking() {
       ga('set', 'anonymizeIp', true);
       ga('send', 'pageview', location.pathname + location.search);
       clearInterval(waitingOnGA);
-      logImpressions();
```

Comment:
@benhalpern, I've reworked GA for the featured story.

This does not work with the new `<Feed />` component. It gets called before the element with the `'featured-story-marker'` ID exists. So now this is handled in the homePageFeed.jsx.